index.rst

```diff
@@ -157,3 +152,7 @@ problem is missing information, and mistakes can always be fixed.
 
 * :ref:`genindex`
 * :ref:`search`
+
+.. raw:: html
+
+   <a class="twitter-timeline" data-width="700" data-height="480" data-theme="dark" href="https://twitter.com/SciCompAalto?ref_src=twsrc%5Etfw">Tweets by SciCompAalto</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
I normally try to prevent going to scicomp.aalto.fi from loading any external, tracking resources (which is why I haven't embedded any youtube videos yet).  Of course it would be really nice to have this, it would basically be our news feed.  Any ideas how to have it without hotloading from twitter?

Reply:
That is the "official" way (according to Twitter of course). I can search if there are sphinx extensions to embed tweets.

Reply:
I tried https://pypi.org/project/sphinxcontrib.twitter/ but the result is the same (embeds the widget from Twitter)